google-cloud-storage/src/main/java/com/google/cloud/storage/Notification.java

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.storage;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import java.util.Map;
+import java.util.Objects;
+
+/**
+ * Google Storage Notification object;
+ *
+ * @see <a href="https://cloud.google.com/storage/docs/json_api/v1/notifications">notifications</a>
+ */
```

Comment:
`Google Storage Notification object;` - is not clear at all. Storage does not have Notifications, Pub/Sub does.